app/controllers/home_controller.rb

```diff
@@ -0,0 +1,3 @@
+class HomeController < ApplicationController
+  def index; end
```

Comment:
- [ ] It will be great to make sure that any requests that are sent to your controller the user is authenticated. What is happening at the moment is that if a user who is not logged in tries to access your controller, the app breaks. This is not a very good approach. So in every controller, you need to add this `before_action :authenticate_user!`

Reply:
@peterrobert, Instead of repeating myself, I placed the `before_action: authenticate_user!` in the 
`ApplicationController < ActionController::Base` since each controller inherit from it.



- [ ] This issue has been resolved :heavy_check_mark: